tests/safeds/ml/classical/classification/test_ada_boost.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from safeds.data.tabular.containers import Table
+from safeds.ml.classical.classification import AdaBoost
+
+
+def test_should_throw_value_error() -> None:
```

Comment:
```suggestion
def test_should_throw_value_error_if_learning_rate_is_non_positive() -> None:
```

I'd suggest to be more explicit here, since there might be other cases where a `ValueError` is raised.

Reply:
`test_should_raise_if_learning_rate_is_non_positive` would also be fine. It's more important to have the trigger in the test name than the type of the error.